Tests/Fakes/FakeCollectionViewModel.swift

```diff
@@ -31,12 +29,8 @@ extension XCTestCase {
         includeHeader: Bool = false,
         includeFooter: Bool = false,
         includeSupplementaryViews: Bool = false,
-        expectDidSelectCell: Bool = false,
-        expectConfigureCell: Bool = false,
-        expectWillDisplay: Bool = false,
-        expectDidEndDisplaying: Bool = false,
-        expectDidHighlight: Bool = false,
-        expectDidUnhighlight: Bool = false
+        expectFields: Set<TestExpectField> = [],
```

Comment:
nit: let's call this `expectationFields`

Reply:
Done.